src/Screen/VideoPreview/VideoPreview.jsx

```diff
@@ -32,27 +35,38 @@ const VideoPreview = () => {
       try {
         const { data } = await axios.get(`/api/video/${id}`);
         setVideo(data.video);
+        const response  = await addToHistory(data.video);
```

Comment:
const {data} = await ...

Reply:
I haven't destructured here because . In the next line , i am sending one more payload data. so playload might take data from the above